dicom_app/dicom_reader.py

```diff
@@ -1,14 +1,45 @@
+import glob
+import logging
+import typing as tp
+
 import pydicom
-from pydicom.data import get_testdata_files
+import numpy as np
+
+
+class DicomReader:
+    def __init__(self):
+        self.last_dir = ''
+        self.image_3d = None
+        self.image_shape = None
+        self.ax_aspect = 0
+        self.sag_aspect = 0
+        self.cor_aspect = 0
```

Comment:
I would use the direct terminology, **axial sagittal coronal**

Reply:
It's good idea, would definitely increase readability.